In [2]:
#Importing libraries
import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt
%matplotlib inline

In [3]:
df=pd.read_excel(r"C:\Users\OM RAJ PANDEY\Desktop\Copy of Type of crime vs literacy.xlsx")
df.head(421)

STATE/UT        DISTRICT  YEAR  MURDER  ATTEMPT TO MURDER  \
0       ANDHRA PRADESH           TOTAL  2001    2602               1555   
1    ARUNACHAL PRADESH           TOTAL  2001      83                 53   
2                ASSAM           TOTAL  2001    1356                481   
3                BIHAR           TOTAL  2001    3643               3419   
4         CHHATTISGARH           TOTAL  2001     880                529   
..                 ...             ...   ...     ...                ...   
415       D & N HAVELI           TOTAL  2012       7                  2   
416        DAMAN & DIU           TOTAL  2012       3                  5   
417           DELHI UT  DELHI UT TOTAL  2012     521                439   
418        LAKSHADWEEP           TOTAL  2012       0                  0   
419         PUDUCHERRY           TOTAL  2012      29                 27   

     CULPABLE HOMICIDE NOT AMOUNTING TO MURDER  RAPE  CUSTODIAL RAPE  \
0                                          136   871               0   
1                                            3    33               0   
2                                           40   817               0   
3                                          250   888               0   
4                                           45   959               0   
..                                         ...   ...             ...   
415                                          0     3               0   
416                                          0     5               0   
417                                         82   706               0   
418                                          0     0               0   
419                                         12    13               0   

     OTHER RAPE  KIDNAPPING & ABDUCTION  ...  HURT/GREVIOUS HURT  \
0           871                    1182  ...               34947   
1            33                      83  ...                 466   
2           817                    1480  ...                5805   
3           888                    2159  ...                7544   
4           959                     207  ...                5477   
..          ...                     ...  ...                 ...   
415           3                      13  ...                  25   
416           5                       3  ...                  17   
417         706                    3970  ...                1747   
418           0                       0  ...                   3   
419          13                      19  ...                 818   

     DOWRY DEATHS  ASSAULT ON WOMEN WITH INTENT TO OUTRAGE HER MODESTY  \
0             420                                               3544     
1               0                                                 78     
2              59                                                850     
3             859                                                562     
4              70                                               1763     
..            ...                                                ...     
415             0                                                  2     
416             0                                                  0     
417           134                                                727     
418             0                                                  1     
419             0                                                  9     

     INSULT TO MODESTY OF WOMEN  CRUELTY BY HUSBAND OR HIS RELATIVES  \
0                          2271                                 5791   
1                             3                                   11   
2                             4                                 1248   
3                            21                                 1558   
4                           161                                  840   
..                          ...                                  ...   
415                           0       

In [4]:
df.columns

Index(['STATE/UT', 'DISTRICT', 'YEAR', 'MURDER', 'ATTEMPT TO MURDER',
       'CULPABLE HOMICIDE NOT AMOUNTING TO MURDER', 'RAPE', 'CUSTODIAL RAPE',
       'OTHER RAPE', 'KIDNAPPING & ABDUCTION',
       'KIDNAPPING AND ABDUCTION OF WOMEN AND GIRLS',
       'KIDNAPPING AND ABDUCTION OF OTHERS', 'DACOITY',
       'PREPARATION AND ASSEMBLY FOR DACOITY', 'ROBBERY', 'BURGLARY', 'THEFT',
       'AUTO THEFT', 'OTHER THEFT', 'RIOTS', 'CRIMINAL BREACH OF TRUST',
       'CHEATING', 'COUNTERFIETING', 'ARSON', 'HURT/GREVIOUS HURT',
       'DOWRY DEATHS', 'ASSAULT ON WOMEN WITH INTENT TO OUTRAGE HER MODESTY',
       'INSULT TO MODESTY OF WOMEN', 'CRUELTY BY HUSBAND OR HIS RELATIVES',
       'IMPORTATION OF GIRLS FROM FOREIGN COUNTRIES',
       'CAUSING DEATH BY NEGLIGENCE', 'OTHER IPC CRIMES', 'TOTAL IPC CRIMES',
       'Literacy'],
      dtype='object')

## ANALYSIS OF TYPE OF CRIME VS EACH STATE VS LITERACY RATE BETWEEN 2001 TO 2011

### In Andhra pradesh murder was increased by 206 i.e 7.9%, attempt to murder was incresed by 674 i.e 43.3%, culpable homicide not amounting to murder is increased by 35 i.e 26%, rape incidents are increased by 571 i.e 65.5%, custodial rapes are 0 , other rapes are increased by 571 i.e 65.5%, kidnapping and abduction are increased by 972 i.e 82.2, kidnapping and abduction of womens and girls are increased by 847 i.e 110.7% , kidnapping and abduction of others are increased by 125 i.e 29.9%, dacoity was decreased by 88 i.e 41.1%, preparation and assembly for dacoity are decreased by 2 i.e 22.2%, robbery was decreased by 29 i.e 4.6% burglary was increased by 1080 i.e 14.9% , theft incidents are increased by 11516 i.e 68.7%, auto theft was increased by 5915 i.e 193.8% , other theft was increased by 5601 i.e 40.8, riots are decreased by 791 i.e 26.3% , criminal breach of trust are increased by 386 i.e 48.9%, cheating is increased by 4357 i.e 80.8% , counterfieting is increased by 80 i.e 55.5%, arson was increased by 149 i.e 17%, hurt/grevious hurt was increased by 19505 i.e 55.8%, dowry deaths was increased by 179 i.e 42.6% assault on women with intent to outrage her modesty are increased by 1305 i.e 36.8% , insult to modesty of women are increased by 1387 i.e 61%, cruelty by husband or his relatives are increased by 7585 i.e 130.9%, importation of girls from foreign countries are decreased to 0 , causing death by negligence are increased by 6573 i.e 88.8%, other ipc crimes are increased by 4048 i.e 11.7%, toal ipc crimes are increased by 59691 i.e 45.8% and literacy was increased by 6.52%.
### In Arunachal pradesh murder was decreased by 18 i.e 21%, attempt to murder was decresed by 24 i.e 45%, culpable homicide not amounting to murder is decreased by 1 i.e 33.3 %, rape incidents are increased by 9 i.e 27.2%, custodial rapes are 0 , other rapes are increased by 9 i.e 27.2%, kidnapping and abduction are increased by 10 i.e 12.04%, kidnapping and abduction of womens and girls are increased by 5 i.e 9.09% , kidnapping and abduction of others are increased by 5 i.e 17.8%, dacoity was decreased by 9 i.e 40.9%, preparation and assembly for dacoity are decreased by 2 i.e 100%, robbery was decreased by 20 i.e 23.8% burglary was decreased by 65 i.e 26.2% , theft incidents are decreased by 25 i.e 6.5%, auto theft was increased by 116 i.e 322.2% , other theft was decreased by 145 i.e 35.6, riots are increased by 25 i.e 104.1% , criminal breach of trust are increased by 13 i.e 31.7%, cheating is increased by 44 i.e 146.6% , counterfieting is decreased by 2 i.e 50%, arson was increased by 21 i.e 161.5%, hurt/grevious hurt was decreased by 87 i.e 18.6%, dowry deaths was 0 i.e 0% assault on women with intent to outrage her modesty are decreased by 27 i.e 34.6% , insult to modesty of women are decreased by 3 i.e 100%, cruelty by husband or his relatives are increased by 7 i.e 63.6%, importation of girls from foreign countries are  0 , causing death by negligence are increased by 98 i.e 98%, other ipc crimes are increased by 4 i.e 0.64%, toal ipc crimes are decreased by 56 i.e 2.39% and literacy was increased by 11.08%.
### In Assam murder was decreased by 53 i.e 3.9%, attempt to murder was incresed by 23 i.e 4.78%, culpable homicide not amounting to murder is increased by 8 i.e 20%, rape incidents are increased by 883 i.e 108.07%, custodial rapes are 0 , other rapes are increased by 883 i.e 108.07%, kidnapping and abduction are increased by 2284 i.e 154.3, kidnapping and abduction of womens and girls are increased by 2122 i.e 198.31% , kidnapping and abduction of others are increased by 162 i.e 39.5%, dacoity was decreased by 227 i.e 42.66%, preparation and assembly for dacoity are decreased by 10 i.e 71.4%, robbery was increased by 150 i.e 21.83% burglary was increased by 904 i.e 37.7% , theft incidents are increased by 2942 i.e 54.8%, auto theft was increased by 235 i.e 67.14% , other theft was increased by 2707 i.e 53.9, riots are decreased by 625 i.e 21.1% , criminal breach of trust are increased by 546 i.e 145.98%, cheating is increased by 920 i.e 160% , counterfieting is increased by 14 i.e 30.43%, arson was increased by 65 i.e 14.7%, hurt/grevious hurt was increased by 501 i.e 8.63%, dowry deaths was increased by 62 i.e 105.08% assault on women with intent to outrage her modesty are increased by 343 i.e 40.35% , insult to modesty of women are increased by 4 i.e 100%, cruelty by husband or his relatives are increased by 3998 i.e 320.3%, importation of girls from foreign countries are increased to 0 to 2 , causing death by negligence are increased by 905 i.e 45.02%, other ipc crimes are increased by 16198 i.e 173.8%, toal ipc crimes are increased by 29837 i.e 80.9% and literacy was increased by 8.89%.
### In Bihar murder was decreased by 445 i.e 12.2%, attempt to murder was decresed by 92 i.e 2.69%, culpable homicide not amounting to murder is increased by 98 i.e 39.2%, rape incidents are increased by 46 i.e 5.18%, custodial rapes are 0 to 1 , other rapes are increased by 45 i.e 5.06%, kidnapping and abduction are increased by 2109 i.e 97.6, kidnapping and abduction of womens and girls are increased by 2532 i.e 488.8% , kidnapping and abduction of others are decreased by 423 i.e 25.7%, dacoity was decreased by 735 i.e 56.9%, preparation and assembly for dacoity are decreased by 42 i.e 28.5%, robbery was decreased by 822 i.e 37.3% burglary was increased by 396 i.e 12.2% , theft incidents are increased by 6591 i.e 67.9%, auto theft was increased by 3991 i.e 380.09% , other theft was increased by 2600 i.e 30.05, riots are increased by 1162 i.e 13.5% , criminal breach of trust are increased by 795 i.e 84.03%, cheating is increased by 2117 i.e 127.60% , counterfieting is decreased by 21 i.e 24.7%, arson was increased by 203 i.e 40.43%, hurt/grevious hurt was increased by 11847 i.e 157.03%, dowry deaths was increased by 554 i.e 64.4% assault on women with intent to outrage her modesty are increased by 228 i.e 40.5% , insult to modesty of women are decreased by 10 i.e 47.6%, cruelty by husband or his relatives are increased by 1049 i.e 67.3%, importation of girls from foreign countries are decreased to 73 i.e 87.95 , causing death by negligence are increased by 3471 i.e 144.2%, other ipc crimes are increased by 19038 i.e 51.92%, toal ipc crimes are increased by 47464 i.e 53.67% and literacy was increased by 14.8%.
### In Chhattisgarh murder was increased by 230 i.e 26.13%, attempt to murder was incresed by 218 i.e 41.2%, culpable homicide not amounting to murder is decreased by 17 i.e 37.7%, rape incidents are increased by 94 i.e 9.8%, custodial rapes are 0 , other rapes are increased by 94 i.e 9.80%, kidnapping and abduction are increased by 265 i.e 128, kidnapping and abduction of womens and girls are increased by 194 i.e 113.45% , kidnapping and abduction of others are increased by 71 i.e 197.22%, dacoity was decreased by 19 i.e 21.83%, preparation and assembly for dacoity are decreased by 12 i.e 63.15%, robbery was increased by 132 i.e 39.05% burglary was decreased by 596 i.e 14.38% , theft incidents are increased by 503 i.e 10.45%, auto theft was increased by 1372 i.e 149.13% , other theft was decreased by 869 i.e 22.3, riots are increased by 63 i.e 7.23% , criminal breach of trust are increased by 26 i.e 16.88%, cheating is increased by 578 i.e 143.7% , counterfieting is increased by 57 i.e 712.5%, arson was increased by 74 i.e 34.4%, hurt/grevious hurt was increased by 5628 i.e 102.7%, dowry deaths was increased by 34 i.e 48.5% assault on women with intent to outrage her modesty are decreased by 109 i.e 6.18% , insult to modesty of women are increased by 13 i.e 8.07%, cruelty by husband or his relatives are decreased by 6 i.e 0.71%, importation of girls from foreign countries are increased to 0 to 2, causing death by negligence are increased by 2167 i.e 314.51%, other ipc crimes are increased by 9433 i.e 59.74%, total ipc crimes are increased by 18758 i.e 48.7% and literacy was increased by 5.58%.
### In Goa murder was increased by 12 i.e 33.33%, attempt to murder was decresed by 8 i.e 26.6%, culpable homicide not amounting to murder is increased by 1 i.e 20%, rape incidents are increased by 17 i.e 141.66%, custodial rapes are 0 , other rapes are increased by 17 i.e 141.6%, kidnapping and abduction are increased by 19 i.e 211.11, kidnapping and abduction of womens and girls are increased by 11 i.e 183.33% , kidnapping and abduction of others are increased by 8 i.e 266.66%, dacoity was decreased by 5 i.e 71.4%, preparation and assembly for dacoity is remain 0, robbery was increased by 1 i.e 4% burglary was decreased by 41 i.e 11.42% , theft incidents are increased by 539 i.e 93.5%, auto theft was increased by 295 i.e 175.5% , other theft was increased by 244 i.e 59.8, riots are increased by 10 i.e 15.6% , criminal breach of trust are increased by 6 i.e 15.3%, cheating is increased by 80 i.e 129.0% , counterfieting is increased by 29 i.e 1450%, arson was increased by 8 i.e 50%, hurt/grevious hurt was increased by 41 i.e23.5 %, dowry deaths was decreased by 1 i.e 50% assault on women with intent to outrage her modesty are increased by 12 i.e 70.58% , insult to modesty of women are increased by 5 i.e 71.4%, cruelty by husband or his relatives are increased by 7 i.e 63.6%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 72 i.e 37.69%, other ipc crimes are increased by 304 i.e 43.61%, total ipc crimes are increased by 1108 i.e 47.3% and literacy was increased by 6.7%.
### In Gujrat murder was decreased by 100 i.e 8.1%, attempt to murder was decresed by 59 i.e 10.9%, culpable homicide not amounting to murder is decreased by 51 i.e 54.2%, rape incidents are increased by 153 i.e 53.4%, custodial rapes are 0 , other rapes are increased by 153 i.e 53.4%, kidnapping and abduction are increased by 616 i.e 61.7, kidnapping and abduction of womens and girls are increased by 585 i.e 68.2% , kidnapping and abduction of others are increased by 31 i.e 21.9%, dacoity was decreased by 88 i.e 41.1%, preparation and assembly for dacoity are decreased by 106 i.e 32.4%, robbery was increased by 377 i.e 38% burglary was decreased by 633 i.e 12.3% , theft incidents are increased by 209 i.e 1.3%, auto theft was increased by 3430 i.e 62.3% , other theft was decreased by 3221 i.e 31.1, riots are decreased by 315 i.e 16.3% , criminal breach of trust are decreased by 409 i.e 23.3%, cheating is decreased by 380 i.e 21.3% , counterfieting is increased by 201 i.e 365.4%, arson was decreased by 186 i.e 41.4%, hurt/grevious hurt was decreased by 3601 i.e 26.1%, dowry deaths was decreased by 37 i.e 55.2% assault on women with intent to outrage her modesty are decreased by 71 i.e 9.39% , insult to modesty of women are decreased by 18 i.e 16.2%, cruelty by husband or his relatives are increased by 2385 i.e 65.0%, importation of girls from foreign countries are decreased to 0 , causing death by negligence are increased by 2903 i.e 99.2%, other ipc crimes are increased by 19069 i.e 37.5%, total ipc crimes are increased by 19952 i.e 19.2% and literacy was increased by 8.9%.
### In Haryana murder was increased by 281 i.e 35.9%, attempt to murder was incresed by 384 i.e 82.2%, culpable homicide not amounting to murder is decreased by 18 i.e 23%, rape incidents are increased by 335 i.e 84.1%, custodial rapes are 0 , other rapes are increased by 335 i.e 84.1%, kidnapping and abduction are increased by 510 i.e 113.5, kidnapping and abduction of womens and girls are increased by 436 i.e 146.8% , kidnapping and abduction of others are increased by 74 i.e 48.6%, dacoity was increased by 90 i.e 116.8%, preparation and assembly for dacoity are increased by 35 i.e 24.8%, robbery was increased by 241 i.e 60.7% burglary was increased by 1902 i.e 61.1% , theft incidents are increased by 11308 i.e 184.8%, auto theft was increased by 9519 i.e 314.9% , other theft was increased by 1789 i.e 57.8, riots are increased by 716 i.e 95.4% , criminal breach of trust are increased by 256 i.e 46%, cheating is increased by 745 i.e 73.3% , counterfieting is decreased by 19 i.e 51.3%, arson was decreased by 37 i.e 18.2%, hurt/grevious hurt was decreased by 1321 i.e 27.8%, dowry deaths was decreased by 30 i.e 10.5% assault on women with intent to outrage her modesty are decreased by 4 i.e 0.8% , insult to modesty of women are increased by 89 i.e 22.1%, cruelty by husband or his relatives are increased by 1227 i.e 81.0%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 649 i.e 75.8%, other ipc crimes are increased by 4643 i.e 29.1%, total ipc crimes are increased by 21982 i.e 56.7% and literacy was increased by 17.6%.
### In Himachal pradesh murder was increased by 11 i.e 9.2%, attempt to murder was decresed by 25 i.e 33.3%, culpable homicide not amounting to murder is decreased by 5 i.e 45.4%, rape incidents are increased by 44 i.e 35.4%, custodial rapes are 0 , other rapes are increased by 44 i.e 35.4%, kidnapping and abduction are increased by 86 i.e 68.2, kidnapping and abduction of womens and girls are increased by 86 i.e 81.9% , kidnapping and abduction of others are same %, dacoity was decreased by 3 i.e 75%, preparation and assembly for dacoity are from 0 to 0, robbery was decreased by 18 i.e 64.2% burglary was increased by 151 i.e 17.8% , theft incidents are increased by 442 i.e 73.6%, auto theft was increased by 214 i.e 150.7% , other theft was increased by 228 i.e 49.7, riots are decreased by 98 i.e 15.6% , criminal breach of trust are increased by 74 i.e 107.2%, cheating is increased by 213 i.e 145.8% , counterfieting is increased by 2 i.e 100%, arson was decreased by 24 i.e 17.9%, hurt/grevious hurt was decreased by 152 i.e 10.8%, dowry deaths was decreased by 6 i.e 60% assault on women with intent to outrage her modesty are increased by 21 i.e 67.7% , insult to modesty of women are increased by 48 i.e 342.8%, cruelty by husband or his relatives are decreased by 78 i.e 24.6%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 80 i.e 16.5%, other ipc crimes are increased by 2050 i.e 33.8%, total ipc crimes are increased by 2813 i.e 24.4% and literacy was increased by 6.3%.
### In Jammu & Kashmir murder was decreased by 906 i.e 84.2%, attempt to murder was decresed by 980 i.e 66.4%, culpable homicide not amounting to murder is decreased by 4 i.e 12.1%, rape incidents are increased by 108 i.e 63.9%, custodial rapes are 0 , other rapes are increased by 108 i.e 63.9%, kidnapping and abduction are increased by 471 i.e 77.7, kidnapping and abduction of womens and girls are increased by 519 i.e 102.9% , kidnapping and abduction of others are decreased by 48 i.e 47%, dacoity was decreased by 10 i.e 41.6%, preparation and assembly for dacoity are from 0 to 0, robbery was decreased by 73 i.e 45.3% burglary was increased by 171 i.e 12.7% , theft incidents are increased by 736 i.e 38.3%, auto theft was increased by 297 i.e 46.9% , other theft was increased by 439 i.e 34.1, riots are increased by 333 i.e 31% , criminal breach of trust are increased by 105 i.e 116.6%, cheating is increased by 221 i.e 56.5% , counterfieting is increased by 24 i.e 160%, arson was decreased by 232 i.e 53%, hurt/grevious hurt was decreased by 72 i.e 22.6%, dowry deaths was decreased by 2 i.e 15.3% assault on women with intent to outrage her modesty are increased by 572 i.e 91.9% , insult to modesty of women are increased by 62 i.e 21.5%, cruelty by husband or his relatives are increased by 236 i.e 472%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 28 i.e 7.8%, other ipc crimes are increased by 4211 i.e 46.5%, total ipc crimes are increased by 4999 i.e 25.6% and literacy was increased by 11.6%.
### In Jharkhand murder was increased by 240 i.e 15.9%, attempt to murder was decresed by 148 i.e 17%, culpable homicide not amounting to murder is decreased by 60 i.e 41.9%, rape incidents are increased by 217 i.e 38.2%, custodial rapes are 0 , other rapes are increased by 217 i.e 38.2%, kidnapping and abduction are increased by 500 i.e 113.3, kidnapping and abduction of womens and girls are increased by 381 i.e 136.5% , kidnapping and abduction of others are increased by 119 i.e 73.4%, dacoity was decreased by 327 i.e 51.4%, preparation and assembly for dacoity are decreased by 8 i.e 16.6%, robbery was decreased by 33 i.e 5.1% burglary was decreased by 31 i.e 2.4% , theft incidents are increased by 2551 i.e 66.6%, auto theft was increased by 1118 i.e 148.2% , other theft was increased by 1433 i.e 46.6, riots are increased by 115 i.e 5.6% , criminal breach of trust are increased by 164 i.e 57.9%, cheating is increased by 353 i.e 63.7% , counterfieting is decreased by 1 i.e 4.3%, arson was decreased by 27 i.e 16.3%, hurt/grevious hurt was increased by 2149 i.e 95.3%, dowry deaths was increased by 65 i.e 29.9% assault on women with intent to outrage her modesty are increased by 20 i.e 6.7% , insult to modesty of women are increased by 2 i.e 40%, cruelty by husband or his relatives are increased by 175 i.e 36.1%, importation of girls from foreign countries are increased by 4 i.e 200 , causing death by negligence are increased by 1308 i.e 311.4%, other ipc crimes are increased by 3163 i.e 36%, total ipc crimes are increased by 10391 i.e 40.8% and literacy was increased by 12.8%.
### In Karnataka murder was increased by 194 i.e 11.9%, attempt to murder was incresed by 362 i.e 24.5%, culpable homicide not amounting to murder is increased by 11 i.e 14.8%, rape incidents are increased by 34 i.e 117%, custodial rapes are 0 , other rapes are increased by 343 i.e 117%, kidnapping and abduction are increased by 836 i.e 149.5, kidnapping and abduction of womens and girls are increased by 444 i.e 163.8% , kidnapping and abduction of others are increased by 392 i.e 136.1%, dacoity was increased by 36 i.e 20.2%, preparation and assembly for dacoity are increased by 363 i.e 1008.3%, robbery was increased by 1276 i.e 150.6% burglary was decreased by 258 i.e 4% , theft incidents are increased by 6966 i.e 54.1%, auto theft was increased by 6210 i.e 154.6% , other theft was increased by 756 i.e 8.5, riots are increased by 579 i.e 8.6% , criminal breach of trust are decreased by 54 i.e 118.4%, cheating is increased by 3172 i.e 118.4% , counterfieting is decreased by 26 i.e 19.8%, arson was decreased by 121 i.e 34.3%, hurt/grevious hurt was decreased by 1990 i.e 8.5%, dowry deaths was increased by 47 i.e 21.3% assault on women with intent to outrage her modesty are increased by 943 i.e 56.6% , insult to modesty of women are remain same i.e 81, cruelty by husband or his relatives are increased by 1957 i.e 111.5%, importation of girls from foreign countries are 0 to 12 , causing death by negligence are increased by 650 i.e 650%, other ipc crimes are increased by 13204 i.e 27.9%, total ipc crimes are increased by 28502 i.e 26.1% and literacy was increased by 8.6%.
### In Kerala murder was decreased by 107 i.e 22.6%, attempt to murder was decresed by 94 i.e 15.2%, culpable homicide not amounting to murder is decreased by 6 i.e 5.4%, rape incidents are increased by 57 i.e 101.4%, custodial rapes are 0 , other rapes are increased by 570 i.e 101.4%, kidnapping and abduction are increased by 116 i.e 63.3, kidnapping and abduction of womens and girls are increased by 124 i.e 127.8% , kidnapping and abduction of others are decreased by 8 i.e 9.3%, dacoity was decreased by 105 i.e 59.6%, preparation and assembly for dacoity are increased by 232 i.e 1784.6%, robbery was increased by 224 i.e 43.3% burglary was decreased by 1473 i.e 32.9% , theft incidents are decreased by 737 i.e 13.5%, auto theft was decreased by 341 i.e 18.99% , other theft was decreased by 396 i.e 10.8, riots are increased by 2646 i.e 32.6% , criminal breach of trust are decreased by 88 i.e 20.5%, cheating is increased by 2399 i.e 87% , counterfieting is decreased by 183 i.e 76.5%, arson was decreased by 318 i.e 41.4%, hurt/grevious hurt was increased by 1918 i.e 9.6%, dowry deaths was decreased by 12 i.e 44.4% assault on women with intent to outrage her modesty are increased by 1814 i.e 93.4% , insult to modesty of women are increased by 492 i.e 607.4%, cruelty by husband or his relatives are increased by 2816 i.e 8109.9%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 0 to 65, other ipc crimes are increased by 58121 i.e 106.5%, total ipc crimes are increased by 68290 i.e 65.7% and literacy was increased by 3.1%.
### In Madhya pradesh murder was increased by 86 i.e 3.5%, attempt to murder was decresed by 530 i.e 18.4%, culpable homicide not amounting to murder is increased by 14 i.e 11.2%, rape incidents are increased by 555 i.e 19.4%, custodial rapes are 0 , other rapes are increased by 555 i.e 19.4%, kidnapping and abduction are increased by 332 i.e 34.7, kidnapping and abduction of womens and girls are increased by 420 i.e 62.8% , kidnapping and abduction of others are decreased by 88 i.e 30.5%, dacoity was decreased by 48 i.e 28.9%, preparation and assembly for dacoity are decreased by 13 i.e 10%, robbery was increased by 188 i.e 10.6% burglary was decreased by 3319 i.e 24.4% , theft incidents are increased by 2151 i.e 10.6%, auto theft was increased by 7060 i.e 155.6% , other theft was decreased by 4909 i.e 31.2, riots are decreased by 1315 i.e 38.7% , criminal breach of trust are increased by 76 i.e 15.8%, cheating is increased by 447 i.e 31.6% , counterfieting is decreased by 7 i.e 36.8%, arson was decreased by 230 i.e 22.6%, hurt/grevious hurt was increased by 4816 i.e 15.5%, dowry deaths was increased by 202 i.e 33.1% assault on women with intent to outrage her modesty are decreased by 398 i.e 5.6% , insult to modesty of women are increased by 11 i.e 1.4%, cruelty by husband or his relatives are increased by 1170 i.e 45.6%, importation of girls from foreign countries are 0 to 45 , causing death by negligence are increased by 4181 i.e 126.8%, other ipc crimes are increased by 26939 i.e 31.6%, total ipc crimes are increased by 35353 i.e 19.4% and literacy was increased by 5.6%.
### In Maharashtra murder was decreased by 21 i.e 0.7%, attempt to murder was incresed by 651 i.e 44.7%, culpable homicide not amounting to murder is increased by 43 i.e 42.5%, rape incidents are increased by 399 i.e 30.6%, custodial rapes are 0 , other rapes are increased by 399 i.e 30.6%, kidnapping and abduction are increased by 684 i.e 69.4, kidnapping and abduction of womens and girls are increased by 641 i.e 104.9% , kidnapping and abduction of others are increased by 43 i.e 11.4%, dacoity was increased by 244 i.e 46.1%, preparation and assembly for dacoity are decreased by 37 i.e 11.2%, robbery was increased by 2010 i.e 89.7% burglary was increased by 1240 i.e 8.2% , theft incidents are increased by 13583 i.e 34.0%, auto theft was increased by 10397 i.e 117.1% , other theft was increased by 3186 i.e 10.2, riots are increased by 1837 i.e 27.3% , criminal breach of trust are increased by 341 i.e 22.1%, cheating is increased by 4825 i.e 112.9% , counterfieting is increased by 86 i.e 32.4%, arson was increased by 47 i.e 3.8%, hurt/grevious hurt was decreased by 443 i.e 1.4%, dowry deaths was decreased by 31 i.e 10% assault on women with intent to outrage her modesty are increased by 971 i.e 34.3% , insult to modesty of women are decreased by 49 i.e 4.3%, cruelty by husband or his relatives are increased by 1046 i.e 17.1%, importation of girls from foreign countries are 1 to 0 , causing death by negligence are increased by 4031 i.e 44.8%, other ipc crimes are increased by 2151 i.e 5%, total ipc crimes are increased by 33669 i.e 19.6% and literacy was increased by 5.4%.
### In Manipur murder was decreased by 131 i.e 62.6%, attempt to murder was incresed by 77 i.e 45.8%, culpable homicide not amounting to murder is decreased by 1 i.e 20%, rape incidents are increased by 33 i.e 165%, custodial rapes are 0 , other rapes are increased by 33 i.e 165%, kidnapping and abduction are increased by 75 i.e 79.7, kidnapping and abduction of womens and girls are increased by 54 i.e 87% , kidnapping and abduction of others are increased by 21 i.e 65.6%, dacoity was decreased by 19 i.e 95%, preparation and assembly for dacoity are increased by 137 i.e 805.8%, robbery was decreased by 12 i.e 63.1% burglary was decreased by 23 i.e 30.6% , theft incidents are increased by 502 i.e 231.3%, auto theft was increased by 482 i.e 678.8% , other theft was increased by 20 i.e 13.6, riots are decreased by 61 i.e 46.5% , criminal breach of trust are increased by 9 i.e 27.2%, cheating is increased by 60 i.e 90.9% , counterfieting is decreased by 8 i.e 40%, arson was increased by 35 i.e 54.6%, hurt/grevious hurt was increased by 91 i.e 45.2%, dowry deaths was increased by 0 to 1, assault on women with intent to outrage her modesty are increased by 17 i.e 80.9% , insult to modesty of women are 0, cruelty by husband or his relatives are increased by 34 i.e 680%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are decreased by 5 i.e 71.4%, other ipc crimes are decreased by 82 i.e 7.4%, total ipc crimes are increased by 729 i.e 29.2% and literacy was increased by 6.4%.
### In Meghalay murder was increased by 6 i.e 3.6%, attempt to murder was incresed by 4 i.e 8.5%, culpable homicide not amounting to murder is increased by 1 i.e 50%, rape incidents are increased by 104 i.e 400%, custodial rapes are 0 , other rapes are increased by 104 i.e 400%, kidnapping and abduction are increased by 32 i.e 58.1%, kidnapping and abduction of womens and girls are increased by 26 i.e 236.3% , kidnapping and abduction of others are increased by 6 i.e 13.6%, dacoity was decreased by 48 i.e 49.4%, preparation and assembly for dacoity are decreased by 3 i.e 100%, robbery was decreased by 62 i.e 49.6% burglary was decreased by 25 i.e 14.7% , theft incidents are increased by 425 i.e 156.8%, auto theft was increased by 174 i.e 1087.5% , other theft was increased by 251 i.e 98.4, riots are increased by 0 i.e 0% , criminal breach of trust are increased by 33 i.e 173.6%, cheating is increased by 137 i.e 472.4% , counterfieting is increased by 10 i.e 333.3%, arson was increased by 8 i.e 30.7%, hurt/grevious hurt was increased by 95 i.e 93.1%, dowry deaths was increased by 0 to 1, assault on women with intent to outrage her modesty are increased by 49 i.e 196% , insult to modesty of women are increased by 0 to 1, cruelty by husband or his relatives are increased by 17 i.e 425%, importation of girls from foreign countries are 0 to 3 , causing death by negligence are increased by 0 to 90, other ipc crimes are increased by 190 i.e 37.1%, total ipc crimes are increased by 1068 i.e 63.3% and literacy was increased by 11.9%.
### In Mizoram murder was decreased by 5 i.e 16.1%, attempt to murder was incresed by 9 i.e 60%, culpable homicide not amounting to murder is increased by 3 i.e 60%, rape incidents are increased by 25 i.e 48%, custodial rapes are 0 , other rapes are increased by 255 i.e 48%, kidnapping and abduction are increased by 4 i.e 200, kidnapping and abduction of womens and girls are decreased by 1 i.e 100% , kidnapping and abduction of others are increased by 5 i.e 500%, dacoity was decreased by 2 i.e 66.6%, preparation and assembly for dacoity are from 0 to 0, robbery was decreased by 17 i.e 73.9% burglary was decreased by 81 i.e 19.4% , theft incidents are decreased by 144 i.e 16.4%, auto theft was increased by 22 i.e 129.4% , other theft was decreased by 166 i.e 19.2, riots are decreased by 3 i.e 100% , criminal breach of trust are decreased by 2 i.e 8.6%, cheating is decreased by 5 i.e 7% , counterfieting is decreased by 10 i.e 83.3%, arson was decreased by 5 i.e 20.8%, hurt/grevious hurt was increased by 3 i.e 3.1%, dowry deaths are from 0 to 0,  assault on women with intent to outrage her modesty are increased by 20 i.e 38.4% , insult to modesty of women are increased by 0 to 1, cruelty by husband or his relatives are decreased by 7 i.e 43.7%, importation of girls from foreign countries are 3 to 0 , causing death by negligence are increased by 14 i.e 60.8%, other ipc crimes are decreased by 220 i.e 44%, total ipc crimes are decreased by 425 i.e 18.9% and literacy was increased by 2.5%.
### In Nagaland murder was decreased by 55 i.e 54.4%, attempt to murder was incresed by 4 i.e 10.2%, culpable homicide not amounting to murder is increased by 2 i.e 22.2%, rape incidents are increased by 6 i.e 35.2%, custodial rapes are 0 , other rapes are increased by 6 i.e 35.2%, kidnapping and abduction are increased by 11 i.e 47.8, kidnapping and abduction of womens and girls are decreased by 3 i.e 50% , kidnapping and abduction of others are increased by 14 i.e 82.3%, dacoity was decreased by 4 i.e 36.3%, preparation and assembly for dacoity are from 0 to 0, robbery was decreased by 70 i.e 54.2% burglary was decreased by 90 i.e 55.2% , theft incidents are increased by 102 i.e 39.5%, auto theft was decreased by 4 i.e 4.4% , other theft was increased by 106 i.e 62.7, riots are decreased by 1 i.e 50% , criminal breach of trust are decreased by 4 i.e 20%, cheating is decreased by 3 i.e 8.8% , counterfieting is increased by 3 i.e 150%, arson was increased by 1 i.e 100%, hurt/grevious hurt was increased by 18 i.e 48.6%, dowry deaths was from 0 to 0 assault on women with intent to outrage her modesty are increased by 3 i.e 50% , insult to modesty of women are from 0 to 0, cruelty by husband or his relatives are increased by 0 to 1, importation of girls from foreign countries are 0 to 0 , causing death by negligence are decreased by 4 i.e 21.0%, other ipc crimes are decreased by 71 i.e 19.5%, total ipc crimes are decreased by 151 i.e 12.2% and literacy was increased by 12.9%.
### In Odisha murder was increased by 490 i.e 49.6%, attempt to murder was incresed by 470 i.e 40.8%, culpable homicide not amounting to murder is decreased by 2 i.e 3.7%, rape incidents are increased by 322 i.e 40.7%, custodial rapes are 0 , other rapes are increased by 322 i.e 40.7%, kidnapping and abduction are increased by 617 i.e 118.1%, kidnapping and abduction of womens and girls are increased by 574 i.e 132.2% , kidnapping and abduction of others are increased by 43 i.e 48.8%, dacoity was increased by 218 i.e 109.5%, preparation and assembly for dacoity are increased by 62 i.e 281.8%, robbery was increased by 625 i.e 65.2% burglary was decreased by 62 i.e 2% , theft incidents are increased by 3102 i.e 55.1%, auto theft was increased by 2169 i.e 265.4% , other theft was increased by 933 i.e 19.4, riots are increased by 130 i.e 9.1% , criminal breach of trust are decreased by 35 i.e 12.2%, cheating is increased by 710 i.e 104.2% , counterfieting is from 10 to 10, arson was from 423 to 423, hurt/grevious hurt was increased by 3527 i.e 80.7%, dowry deaths was increased by 171 i.e 58.1% assault on women with intent to outrage her modesty are increased by 1552 i.e 93.7% , insult to modesty of women are decreased by 233 i.e 48.6%, cruelty by husband or his relatives are increased by 1054 i.e 83.2%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 2189 i.e 137.7%, other ipc crimes are decreased by 301 i.e 1.4%, total ipc crimes are increased by 14616 i.e 31.3% and literacy was increased by 9.7%.
### In Punjab murder was increased by 104 i.e 14%, attempt to murder was incresed by 208 i.e 26.3%, culpable homicide not amounting to murder is increased by 32 i.e 40%, rape incidents are increased by 181 i.e 60.7%, custodial rapes are 0 , other rapes are increased by 181 i.e 60.7%, kidnapping and abduction are increased by 168 i.e 32.7%, kidnapping and abduction of womens and girls are increased by 193 i.e 59.5% , kidnapping and abduction of others are decreased by 25 i.e 13.2%, dacoity was decreased by 17 i.e 37.7%, preparation and assembly for dacoity are increased by 59 i.e 70.2%, robbery was increased by 105 i.e 80.1% burglary was increased by 586 i.e 30.8% , theft incidents are increased by 1761 i.e 58.2%, auto theft was increased by 903 i.e 75% , other theft was increased by 858 i.e 47.1, riots are decreased by 1 i.e 100% , criminal breach of trust are decreased by 87 i.e 23.6%, cheating is increased by 950 i.e 36.2% , counterfieting is from 58 to 58, arson was increased by 20 i.e 31.7%, hurt/grevious hurt was increased by 761 i.e 19%, dowry deaths was decreased by 16 i.e 10% assault on women with intent to outrage her modesty are decreased by 90 i.e 24.1% , insult to modesty of women are decreased by 16 i.e 34%, cruelty by husband or his relatives are increased by 8 i.e 0.7%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 1422 i.e 66%, other ipc crimes are increased by 971 i.e 10.5%, total ipc crimes are increased by 7109 i.e 25.5% and literacy was increased by 6.14%.
### In Rajasthan murder was increased by 202 i.e 16%, attempt to murder was decresed by 357 i.e 18.5%, culpable homicide not amounting to murder is increased by 37 i.e 58.7%, rape incidents are increased by 751 i.e 71.5%, custodial rapes are 0 , other rapes are increased by 751 i.e 71.5%, kidnapping and abduction are increased by 486 i.e 17.8, kidnapping and abduction of womens and girls are increased by 548 i.e 25.3% , kidnapping and abduction of others are decreased by 62 i.e 11.2%, dacoity was decreased by 32 i.e 53.3%, preparation and assembly for dacoity are increased by 32 i.e 80%, robbery was decreased by 162 i.e 18.2% burglary was decreased by 2774 i.e 38% , theft incidents are increased by 4160 i.e 24.5%, auto theft was increased by 8242 i.e 190.6% , other theft was decreased by 4082 i.e 32%, riots are decreased by 10463 i.e 93.3% , criminal breach of trust are decreased by 285 i.e 27.7%, cheating is increased by 10853 i.e 163.3% , counterfieting is decreased by 33 i.e 37%, arson was decreased by 619 i.e 56.6%, hurt/grevious hurt was decreased by 9508 i.e 34.5%, dowry deaths was increased by 138 i.e 36.7% assault on women with intent to outrage her modesty are decreased by 431 i.e 14.9% , insult to modesty of women are decreased by 47 i.e 83.9%, cruelty by husband or his relatives are increased by 6686 i.e 120.8%, importation of girls from foreign countries are 1 to 0 , causing death by negligence are increased by 3901 i.e 92.4%, other ipc crimes are increased by 7903 i.e 12.6%, total ipc crimes are increased by 10437 i.e 6.7% and literacy was increased by 6.01%.
### In Sikkim  murder was decreased by 5 i.e 26.3%, attempt to murder was decresed by 4 i.e 36.3%, culpable homicide not amounting to murder is increased by 1 i.e 14.2%, rape incidents are increased by 8 i.e 100%, custodial rapes are 0 , other rapes are increased by 8 i.e 100%, kidnapping and abduction are increased by 4 i.e 66.6, kidnapping and abduction of womens and girls are increased by 8 i.e 400% , kidnapping and abduction of others are decreased by 4 i.e 100%, dacoity was increased by 0 to 0 , preparation and assembly for dacoity are increased by 0 to 0, robbery was increased by 2 i.e 66.6% burglary was decreased by 16 i.e 21% , theft incidents are decreased by 2 i.e 2.7%, auto theft was increased by 2 i.e 33.3% , other theft was decreased by 4 i.e 5.8, riots are increased by 22 i.e 733.3% , criminal breach of trust are increased by 3 i.e 300%, cheating is increased by 18 i.e 138.4% , counterfieting is increased by 3 , arson was increased by 3 i.e 300%, hurt/grevious hurt was increased by 40 i.e 114.2%, dowry deaths was from 0 to 0, assault on women with intent to outrage her modesty are increased by 24  , insult to modesty of women are decreased by 14 i.e 100%, cruelty by husband or his relatives are increased by 0 to 4%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 50 i.e 714.2%, other ipc crimes are increased by 11 i.e 6.62%, total ipc crimes are increased by 152 i.e 34.2% and literacy was increased by 12.6%.
### In Tamil nadu murder was increased by 215 i.e 12.9%, attempt to murder was incresed by 758 i.e 34.3%, culpable homicide not amounting to murder is increased by 9 i.e 47.3%, rape incidents are increased by 254 i.e 60%, custodial rapes are 0 , other rapes are increased by 254 i.e 60%, kidnapping and abduction are increased by 1032 i.e 108.4, kidnapping and abduction of womens and girls are increased by 1136 i.e 187.1% , kidnapping and abduction of others are decreased by 104 i.e 30.1%, dacoity was decreased by 57 i.e 36%, preparation and assembly for dacoity are increased by 5 i.e 83.3%, robbery was increased by 1394 i.e 207.4% burglary was decreased by 1117 i.e 18.7% , theft incidents are decreased by 2785 i.e 16.6%, auto theft was increased by 443 i.e 9.6% , other theft was decreased by 3218 i.e 26.3, riots are decreased by 2187 i.e 42% , criminal breach of trust are decreased by 122 i.e 33.8%, cheating is increased by 1733 i.e 81% , counterfieting is increased by 171 i.e 164.4%, arson was decreased by 274 i.e 27.9%, hurt/grevious hurt was decreased by 7262 i.e 25.5%, dowry deaths was decreased by 39 i.e 20.4% assault on women with intent to outrage her modesty are decreased by 306 i.e 17.2% , insult to modesty of women are decreased by 548 i.e 54.1%, cruelty by husband or his relatives are increased by 997 i.e 122.3%, importation of girls from foreign countries are 14 to 0 , causing death by negligence are increased by 7526 i.e 80%, other ipc crimes are increased by 38695 i.e 50.6%, total ipc crimes are increased by 38078 i.e 24.5% and literacy was increased by 6.5%.
### In Tripura murder was decreased by 77 i.e 32%, attempt to murder was incresed by 50 i.e 200%, culpable homicide not amounting to murder is  from 0 to 0, rape incidents are increased by 103 i.e 100.9%, custodial rapes are 0 , other rapes are increased by 103 i.e 100.9%, kidnapping and abduction are increased by 61 i.e 65.5, kidnapping and abduction of womens and girls are increased by 81 i.e 231.4% , kidnapping and abduction of others are decreased by 20 i.e 34.4%, dacoity was decreased by 15 i.e 57.6%, preparation and assembly for dacoity are from 0 to 0 , robbery was decreased by 13 i.e 20.6% burglary was increased by 29 i.e 14.6% , theft incidents are increased by 235 i.e 90.7%, auto theft was increased by 53 i.e 883.3% , other theft was increased by 182 i.e 71.9, riots are increased by 37 i.e 26.8% , criminal breach of trust are increased by 50 i.e 1250%, cheating is increased by 82 i.e 482.3% , counterfieting is increased by 3 i.e 50%, arson was increased by 27 i.e 87%, hurt/grevious hurt was increased by 928 i.e 225.2%, dowry deaths was increased by 14 i.e 87.5% assault on women with intent to outrage her modesty are increased by 236 i.e 406.8% , insult to modesty of women are increased by 9 i.e 9%, cruelty by husband or his relatives are increased by 475 i.e 209.2%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are decreased by 17 i.e 54.8%, other ipc crimes are increased by 785 i.e 91.8%, total ipc crimes are increased by 3002 i.e 107.1% and literacy was increased by 14%
### In Uttar pradesh murder was decreased by 3311 i.e 41.5%, attempt to murder was decresed by 3311 i.e 41.5%, culpable homicide not amounting to murder is increased by 268 i.e 22.5%, rape incidents are increased by 84 i.e 4.2%, custodial rapes are 0 , other rapes are increased by 84 i.e 4.2%, kidnapping and abduction are increased by 4170 i.e 96.3, kidnapping and abduction of womens and girls are increased by 4646 i.e 161.3% , kidnapping and abduction of others are decreased by 476 i.e 32.8%, dacoity was decreased by 526 i.e 58.1%, preparation and assembly for dacoity are increased by 1 i.e 2.6%, robbery was decreased by 677  i.e 17.6% burglary was decreased by 2646 i.e 31.4% , theft incidents are increased by 9672 i.e 35.8%, auto theft was increased by 15174 i.e 245.1% , other theft was decreased by 5502 i.e 26.4, riots are decreased by 2321 i.e 31.6% , criminal breach of trust are increased by 291 i.e 8.4%, cheating is increased by 4578 i.e 85.3% , counterfieting is increased by 123 i.e 83.6%, arson was decreased by 335 i.e 41.2%, hurt/grevious hurt was decreased by 5582 i.e 31.3%, dowry deaths was increased by 111 i.e 5% assault on women with intent to outrage her modesty are increased by 585 i.e 20.3% , insult to modesty of women are decreased by 2572 i.e 99.8%, cruelty by husband or his relatives are decreased by 244 i.e 3.3%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 7251 i.e 101.7%, other ipc crimes are increased by 10736 i.e 18.5%, total ipc crimes are increased by 17006 i.e 9.54% and literacy was increased by 11.3%.
### In Uttarakhand murder was decreased by 138 i.e 43.6%, attempt to murder was decresed by 90 i.e 32.2%, culpable homicide not amounting to murder is increased by 16 i.e 42.1 %, rape incidents are increased by 55 i.e 74.3%, custodial rapes are 0 , other rapes are increased by 55 i.e 74.3%, kidnapping and abduction are increased by 131 i.e 71.5, kidnapping and abduction of womens and girls are increased by 157 i.e 124.6% , kidnapping and abduction of others are decreased by 26 i.e 45.6%, dacoity was decreased by 19 i.e 59.3%, preparation and assembly for dacoity are decreased by 2 i.e 66.6%, robbery was decreased by 12 i.e 6.2% burglary was decreased by 191 i.e 35.8% , theft incidents are increased by 674 i.e 47.4%, auto theft was increased by 466 i.e 124.5% , other theft was increased by 208 i.e 19.9, riots are decreased by 39 i.e 8.8% , criminal breach of trust are decreased by 56 i.e 32.7%, cheating is increased by 201 i.e 63.2% , counterfieting is increased by 22 i.e 550%, arson was decreased by 13 i.e 43.3%, hurt/grevious hurt was decreased by 22 i.e 2.6%, dowry deaths was increased by 27 i.e 48.2% assault on women with intent to outrage her modesty are increased by 13 i.e 12.6% , insult to modesty of women are decreased by 12 i.e 14.2%, cruelty by husband or his relatives are increased by 6 i.e 1.9%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 255 i.e 57.8%, other ipc crimes are decreased by 149 i.e 6.6%, total ipc crimes are increased by 701 i.e 8.6% and literacy was increased by 7.2%.
### In West Bengal murder was increased by 515 i.e 32.3%, attempt to murder was incresed by 1793 i.e 399.3%, culpable homicide not amounting to murder is decreased by 95 i.e 16.3%, rape incidents are increased by 1654 i.e 233.2 %, custodial rapes are 0 , other rapes are increased by 1654 i.e 233.2%, kidnapping and abduction are increased by 3410 i.e 389.7%, kidnapping and abduction of womens and girls are increased by 3016 i.e 433.9% , kidnapping and abduction of others are increased by 394 i.e 218.8%, dacoity was decreased by 38 i.e 13.8%, preparation and assembly for dacoity are increased by 518 i.e 123%, robbery was increased by 100 i.e 15.1% burglary was increased by 29 i.e 6.8% , theft incidents are increased by 7201 i.e 50.5%, auto theft was increased by 2080 i.e 81.1% , other theft was increased by 5121 i.e 43.8, riots are increased by 2924 i.e 94.4% , criminal breach of trust are increased by 778 i.e 160%, cheating is increased by 2995 i.e 243% , counterfieting is decreased by 105 i.e 102.9%, arson was decreased by 311 i.e 304.9%, hurt/grevious hurt was increased by 9436 i.e 257.9%, dowry deaths was increased by 245 i.e 92.4% assault on women with intent to outrage her modesty are increased by 1409 i.e 147.6% , insult to modesty of women are increased by 152 i.e 316.6%, cruelty by husband or his relatives are increased by 15913 i.e 412.3%, importation of girls from foreign countries are 3 to 0 , causing death by negligence are increased by 968 i.e 42.4%, other ipc crimes are increased by 31354 i.e 124.1%, total ipc crimes are increased by 81634 i.e 132.6% and literacy was increased by 7.6%.
### In A & N islands murder was increased by 1 i.e 7.6%, attempt to murder was incresed by 6 , culpable homicide not amounting to murder is increased by 2 , rape incidents are increased by 10 i.e 333.3%, custodial rapes are 0 , other rapes are increased by 10 i.e 333.3%, kidnapping and abduction are increased by 13 i.e 650%, kidnapping and abduction of womens and girls are increased by 10 i.e 500% , kidnapping and abduction of others are increased by 0 to 3, dacoity was increased by 0 to 1, preparation and assembly for dacoity are increased by 0 to 0, robbery was decreased by 4 to 1 burglary was increased by 35 i.e 54.6% , theft incidents are increased by 23 i.e 35.3%, auto theft was from 4 to 4 , other theft was increased by 23 i.e 37, riots are decreased by 2 i.e 15.3% , criminal breach of trust are decreased by 7 i.e 70%, cheating is increased by 24 i.e 300% , counterfieting is decreased by 2 to 0, arson was increased by 2 i.e 50%, hurt/grevious hurt was decreased by 37 i.e 31.3%, dowry deaths was from 0 to 0, assault on women with intent to outrage her modesty are decreased by 4 i.e 21% , insult to modesty of women are increased by 2 i.e 200%, cruelty by husband or his relatives are decreased by 4 i.e 44%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 0 to 6, other ipc crimes are increased by 69 i.e 21.3%, total ipc crimes are increased by 135 i.e 20.5% and literacy was increased by 5.3%.
### In Chandigarh murder was increased by 9 i.e 60%, attempt to murder was incresed by 30 i.e 300%, culpable homicide not amounting to murder is from 6 to 6, rape incidents are increased by 9 i.e 50%, custodial rapes are 0 , other rapes are increased by 9 i.e 50%, kidnapping and abduction are increased by 2 i.e 3.5, kidnapping and abduction of womens and girls are decreased by 4 i.e 8% , kidnapping and abduction of others are increased by 6 i.e 100%, dacoity was increased by 1 i.e 20%, preparation and assembly for dacoity are increased by 0 to 2, robbery was increased by 46 i.e 209.9% burglary was decreased by   79 i.e 21.7% , theft incidents are increased by 27 i.e 1.7%, auto theft was increased by 421 i.e 85.2% , other theft was increased by 448 i.e 43.2, riots are decreased by 25 i.e 27.4% , criminal breach of trust are decreased by 1 i.e 3.8%, cheating is increased by 100 i.e 64.5% , counterfieting is decreased by 2 i.e 100%, arson was increased by 4 i.e 80%, hurt/grevious hurt was decreased by 50 i.e 52.6%, dowry deaths was decreased by 1 i.e 33.3% assault on women with intent to outrage her modesty are decreased by 3 i.e 12.5% , insult to modesty of women are decreased by 3 i.e 20%, cruelty by husband or his relatives are increased by 10 i.e 27.7%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 1 i.e 16.6%, other ipc crimes are increased by 122 i.e 13.3%, total ipc crimes are increased by 145 i.e 4.2% and literacy was increased by 4.1%.
### In D & N Haveli murder was increased by 11 i.e 366.6%, attempt to murder was from 2 to 2, culpable homicide not amounting to murder is from 0 to 0, rape incidents are decreased by 2 i.e 33.3%, custodial rapes are 0 , other rapes are decreased by 2 i.e 33.3%, kidnapping and abduction are increased by 5 i.e 125, kidnapping and abduction of womens and girls are increased by 6 i.e 300% , kidnapping and abduction of others are decreased by 1 i.e 50%, dacoity was increased by 0 to 7, preparation and assembly for dacoity are from 0 to 0 , robbery was increased from 0 to 0 burglary was decreased by 12 i.e 35.2% , theft incidents are increased by 24 i.e 53.3%, auto theft was increased by 28 i.e 186.6% , other theft was decreased by 4 i.e 13.3, riots are increased by 16 i.e 400% , criminal breach of trust are decreased by 3 i.e 23%, cheating is increased by 6 i.e 50% , counterfieting is increased by 1 i.e 100%, arson was increased by 4 i.e 400%, hurt/grevious hurt was decreased by 4 i.e 13.7%, dowry deaths was from 0 to 0, assault on women with intent to outrage her modesty are decreased by 5 i.e 71.4% , insult to modesty of women are from 0 to 0, cruelty by husband or his relatives are decreased by 1 i.e 25%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 0 to 24, other ipc crimes are decreased by 49 i.e 26.7%, total ipc crimes are increased by 22 i.e 6.2% and literacy was increased by 18.6%.
### In Daman & Diu murder was decreased by 1 i.e 14.2%, attempt to murder was decresed by 4 i.e 80%, culpable homicide not amounting to murder is from 0 to 0, rape incidents are increased by 0 to 1, custodial rapes are 0 , other rapes are increased by 0 to 1, kidnapping and abduction are decreased by 2 i.e 40%, kidnapping and abduction of womens and girls are decreased by 1 i.e 33.3% , kidnapping and abduction of others are decreased by 1 i.e 50%, dacoity was increased by 0 to 4, preparation and assembly for dacoity are from 0 to 0, robbery was increased by 0 to 6, burglary was decreased by 12 i.e 27.9% , theft incidents are increased by 7 i.e 17.5%, auto theft was increased by 13 i.e 65% , other theft was decreased by 6 i.e 30, riots are decreased by 4 i.e 17.3% , criminal breach of trust are decreased by 2 i.e 40%, cheating is increased by 1 i.e 11.1% , counterfieting is increased by 1 i.e 100%, arson was decreased by 3 i.e 100%, hurt/grevious hurt was decreased by 10 i.e 47.6%, dowry deaths was from 0 to 0, assault on women with intent to outrage her modesty are from 0 to 0, insult to modesty of women are increased by 0 to 0, cruelty by husband or his relatives are decreased by 2 i.e 50%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 0 to 43, other ipc crimes are decreased by 38 i.e 52%, total ipc crimes are increased by 15 i.e 6.2% and literacy was increased by 8.9%.
### In Delhi UT murder was decreased by 4 i.e 0.7%, attempt to murder was decresed by 124 i.e 24.3%, culpable homicide not amounting to murder is increased by 8 i.e 12.6%, rape incidents are increased by 191 i.e 50.1%, custodial rapes are 0 , other rapes are increased by 191 i.e 50.1%, kidnapping and abduction are increased by 2140 i.e 131.5, kidnapping and abduction of womens and girls are increased by 1121 i.e 116.2% , kidnapping and abduction of others are increased by 1019 i.e 153.6%, dacoity was decreased by 15 i.e 31.2%, preparation and assembly for dacoity are decreased by 49 i.e 66.2%, robbery was decreased by 62 i.e 9.9%, burglary was decreased by 1610 i.e 53.1% , theft incidents are increased by 3623 i.e 18.7%, auto theft was increased by 6774 i.e 85.8% , other theft was decreased by 3151 i.e 27.6, riots are decreased by 115 i.e 69.6% , criminal breach of trust are decreased by 142 i.e 29.6%, cheating is increased by 407 i.e 18.6% , counterfieting is increased by 2 i.e 4.7%, arson was decreased by 8 i.e 16%, hurt/grevious hurt was decreased by 65 i.e 3.2%, dowry deaths was increased by 29 i.e 25.6% assault on women with intent to outrage her modesty are increased by 155 i.e 30.8% , insult to modesty of women are increased by 72 i.e 80%, cruelty by husband or his relatives are increased by 1437 i.e 1041.3%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 736 i.e 170.3%, other ipc crimes are decreased by 7637 i.e 34.7%, total ipc crimes are decreased by 1031 i.e 1.8% and literacy was increased by 4.5%.
### In Lakshdweep murder was decreased by 1 to 0, attempt to murder was incresed by 0 to 1, culpable homicide not amounting to murder is from 0 to 0, rape incidents are increased by 0 to 0%, custodial rapes are 0 , other rapes are from 0 to 0 , kidnapping and abduction are from 0 to 0, kidnapping and abduction of womens and girls are from 0 to 0 , kidnapping and abduction of others are from 0 to 0, dacoity was from 0 to 0, preparation and assembly for dacoity are from 0 to 0, robbery was from 0 to 0, burglary was ddecreased by 1 to 0 , theft incidents are decreased by 10 to 4, auto theft was from 0 to 0 , other theft was decreased by 10 to 4, riots are increased by 4 i.e 400% , criminal breach of trust are from 1 to 1, cheating is from 0 to 0 , counterfieting is from 0 to 0, arson was increased by 2 to 3, hurt/grevious hurt was from 3 to 3, dowry deaths was from 0 to 0, assault on women with intent to outrage her modesty are from 0 to 0 , insult to modesty of women are from 0 to 0, cruelty by husband or his relatives are from 0 to 0, importation of girls from foreign countries are 0 to 0 , causing death by negligence are from 0 to 0, other ipc crimes are increased by 10 i.e 58.8%, total ipc crimes are increased by 8 i.e 22.2% and literacy was increased by 5.1%.
### In Puducherry murder was increased by 7 i.e 28%, attempt to murder was decresed by 10 i.e 31.2%, culpable homicide not amounting to murder is increased by 11 i.e 1100%, rape incidents are decreased by 2 i.e 22.2%, custodial rapes are 0 , other rapes are decreased by 2 i.e 22.2%, kidnapping and abduction are increased by 8 i.e 200%, kidnapping and abduction of womens and girls are increased by 6 i.e 200% , kidnapping and abduction of others are increased by 2 i.e 200%, dacoity was increased by 4 i.e 400%, preparation and assembly for dacoity are increased by 0 to 10, robbery was increased by 7 i.e 175% burglary was decreased by 46 i.e 41.4% , theft incidents are increased by 179 i.e 33.9%, auto theft was increased by 227 i.e 79.9% , other theft was decreased by 48 i.e 19.6, riots are increased by 25 i.e 25% , criminal breach of trust are increased by 4 i.e 80%, cheating is increased by 31 i.e 134.7% , counterfieting is increased by 1 i.e 33.3%, arson was decreased by 21 i.e 67.7%, hurt/grevious hurt was increased by 218 i.e 34.3%, dowry deaths was from 1 to 1, assault on women with intent to outrage her modesty are from 35 to 35 , insult to modesty of women are decreased by 11 i.e 40.7%, cruelty by husband or his relatives are increased by 7 i.e 233.3%, importation of girls from foreign countries are 0 to 0 , causing death by negligence are increased by 77 i.e 46.3%, other ipc crimes are decreased by 205 i.e 8.8%, total ipc crimes are increased by 294 i.e 7.22% and literacy was increased by 4.6%.









.


















